src/components/Authentication/Authentication.js

```diff
@@ -1,10 +1,56 @@
+import { useEffect, useState } from 'react';
 import { useAuth0 } from '@auth0/auth0-react';
 
 import HeaderButton from '../HeaderButton/HeaderButton';
 
+const AUTH0_USER_DOMAIN = process.env.REACT_APP_AUTH0_DOMAIN;
+
 const Authentication = ({ style }) => {
-  const { error, isLoading, isAuthenticated, user, loginWithRedirect, logout } = useAuth0();
+  const {
+    error,
+    isLoading,
+    isAuthenticated,
+    user,
+    getAccessTokenSilently,
+    loginWithRedirect,
+    logout,
+  } = useAuth0();
+
+  const [userMetaData, setUserMetaData] = useState(null);
+
+  useEffect(() => {
+      const getUserMetaData = async () => {
```

Comment:
The data for the user profile must be obtained from the back-end. Profile data will not always be the same as the data from auth0.